scripts/check-codestyle.js

```diff
@@ -119,18 +114,15 @@ const spdx = '// SPDX-License-Identifier: Apache-2.0';
 
 let allYears = [];
 
-allFiles().forEach(file => {
+allFiles().forEach((file) => {
```

Comment:
Not sure what your lint setting but it does not seem to match what we usually have. Can we remove these changes?

Reply:
Sure, will remove all these `()`